spec/lib/zxcvbn_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+require "#{Rails.root}/lib/zxcvbn_tester"
```

Comment:
benefit of doing that is auto-require. downside is that it is required everywhere. more stuff loaded == slower tests. But I am Ok with either way :)